kudo-nginx-operator/manifests/support-bundle.yaml

```diff
@@ -0,0 +1,32 @@
+apiVersion: troubleshoot.replicated.com/v1beta1
```

Comment:
Not a blocker, but I'd encourage to make these more minimal, at least removing things like these defaults, if not making them actually pull the right logs

```
    - secret:
        name: illmannered-cricket-mysql
        namespace: default
        key: mysql-password
    - logs:
        selector:
          - name=nginx-ingress-microk8s
        namespace: default
        limits:
          maxAge: 30d
          maxLines: 10000
```

Reply:
Yeah, it's a good point. My process is to pull from the default yaml and work from that. I think we should maybe make that simpler.